5.0/en/0x52-V52-Tokens.md

```diff
@@ -8,15 +8,15 @@ The use of self-contained tokens has become very widespread, even outside of OID
 
 ## V52.1 Token source and integrity
 
-Before inspecting the contents of a self-contained token, it is necessary to ensure that the token has been produced by a trusted party and that it has not been tampered with.
+This section describes requirements about the content of the token both for the issuer and for the consumer of the token in order to prevent unintended consumption of the token (e.g., by the wrong service, for the wrong usage). Before making security decisions based on the content of a self-contained token, the consumer must validate that the token is intended to be consumed by this consumer and was issued for the purpose for which it was presented. On the other hand, the issuer must associate claims to the token in order to make this validation possible and ensure that a malicious actor may not deceive it into including malicious claims in the token.
```

Comment:
I'm not sure this change is necessary. Some of the text doesn't seem right for this particular section and feels like it repeats what is in 52.2.

I would probably suggest something simple for this section like:

"_This section includes requirements that ensure that the token has been produced by a trusted party and that it has not been tampered with._"

Is there anything else which is super important to add to that?

Reply:
We need to avoid overlap with V52.2 here. Intended usage feels like V52.2 area.

Reply:
I used proposed short text by Josh and move on at the moment. If there is a need or reason to improve it, we can do it again.